src/collective/lazysizes/tests/test_upgrades.py

```diff
@@ -47,7 +47,7 @@ def setUp(self):
 
     def test_upgrade_to_2_registrations(self):
         version = self.setup.getLastVersionForProfile(self.profile_id)[0]
-        self.assertGreaterEqual(version, self.to_version)
+        self.assertGreaterEqual(int(version), int(self.to_version))
```

Comment:
The string in int cast is necessary. The tests passed before this change for metadata version 9 only by the way strings are compared in python. See below:

Comparison of "10" with other numbers-as-strings:
```
>>> "10" >= "0"
True
>>> "10" >= "1"
True
>>> "10" >= "2"
False
>>> "10" >= "3"
False
>>> "10" >= "4"
False
>>> "10" >= "5"
False
>>> "10" >= "6"
False
>>> "10" >= "7"
False
>>> "10" >= "8"
False
>>> "10" >= "9"
False
>>> "10" >= "10"
True
```

Comparison of "9" with other numbers-as-strings:
```
>>> "9" >= "0"
True
>>> "9" >= "1"
True
>>> "9" >= "2"
True
>>> "9" >= "3"
True
>>> "9" >= "4"
True
>>> "9" >= "5"
True
>>> "9" >= "6"
True
>>> "9" >= "7"
True
>>> "9" >= "8"
True
>>> "9" >= "9"
True
>>> "9" >= "10"
True
```